java/src/main/java/org/rocksdb/SstFileWriter.java

```diff
@@ -45,6 +45,13 @@ public SstFileWriter(final EnvOptions envOptions, final Options options) {
         envOptions.nativeHandle_, options.nativeHandle_));
   }
 
+  /**
+   * SstFileWriter Constructor.
+   */
+  public SstFileWriter() {
+    this(new EnvOptions(), new Options());
```

Comment:
You can't just create an `Options` in this way, otherwise it will never be cleaned up (deleted).

Reply:
@adamretter thanks for you point it out. It seem we could not support a constructor without argument ?

Reply:
That's right, you cannot have a zero-arg constructor here. You need at least `EnvOptions` and `Options` as set out by the underlying C++ API.